modules/DestinationTableManager.py

```diff
@@ -114,22 +114,16 @@ def upsert_table(self, schema_name, source_table_name, target_table_name, column
         primary_key_column_list = ','.join(map(str, primary_key_column_array))
 
         sql_builder = io.StringIO()
-        sql_builder.write(f"INSERT INTO {schema_name}.{target_table_name} ({column_list})")
-        sql_builder.write(os.linesep)
-        sql_builder.write(f" SELECT {column_list} FROM {schema_name}.{source_table_name}")
-        sql_builder.write(os.linesep)
+        sql_builder.write(f"INSERT INTO {schema_name}.{target_table_name} ({column_list}) \n")
```

Comment:
should this be changed to f-strings to keep with convention

Reply:
tried. but keeping it the way it is with `.format()` and `io.StringIO` as it looks simpler.